other/edge-server/README.md

```diff
@@ -372,7 +395,7 @@ the React client. The client does not have a listener observing changes
 to the collection. You can manually reload the React client to view the
 new `Item`.
 
-If you reload the React client and it still doesn't show the new `Item`, 
+If you reload the React client and it still doesn't show the new `Item`,
 check:
 
 - The Edge Server is running
```

Comment:
We've added back the `owner_id` here, but it's not reflected in the schema below. We should update this schema to match the one we're now using in the backend.